diff -ur linux-4.19.284.orig/Makefile linux-4.19.284/Makefile
--- linux-4.19.284.orig/Makefile	2023-05-30 21:42:15.000000000 +1000
+++ linux-4.19.284/Makefile	2023-06-01 23:48:53.745728965 +1000
@@ -1202,7 +1202,7 @@
 archscripts:
 
 PHONY += __headers
-__headers: $(version_h) scripts_basic uapi-asm-generic archheaders archscripts
+__headers: $(version_h) scripts_basic uapi-asm-generic archheaders
 	$(Q)$(MAKE) $(build)=scripts build_unifdef
 
 PHONY += headers_install_all
